tests/client/mod.rs

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: Apache-2.0
+
+mod util;
+
+use util::{enarx, run};
+
+/// Just a nice wrapper over `format!` for testing CLI invocations
+macro_rules! cmd {
+    ($($arg:tt)+) => (
+        enarx(format!($($arg)+))
+    )
+}
+
+#[async_std::test]
+async fn full() {
+    run(|oidc_addr, db_addr| {
+        // test for failure when looking up a user that does not exist
+        let cmd = cmd!(
+            "enarx user info --insecure-auth-token-file tests/client/token {db_addr}+testuser"
```

Comment:
Why `+` and not just `{db_addr}/testuser`? That aligns better with existing tooling, e.g. `docker` (same for all other occurences)

Reply:
I also prefer that, I just didn't feel like duplicating all the slug parsing code on the Enarx side, since that's required to support this. I can do that.